op-chain-ops/cmd/celo-migrate/state.go

```diff
@@ -150,6 +151,11 @@ func applyStateMigrationChanges(config *genesis.DeployConfig, genesis *core.Gene
 	// uncle blocks, or receipts in the Cel2 transition block.
 	cel2Block := types.NewBlock(cel2Header, nil, nil, nil, trie.NewStackTrie(nil))
 
+	// Create genesis block for new chains to sync
+	syncGenesis := AlfajoresGenesisBlockFromConfig(cfg)
```

Comment:
We need to check the chain id and use the appropriate genesis (i.e mainnet, baklava or alfajores) if it's not one of those then we can't write the genesis.